etc/ci/update_nightlies.sh

```diff
@@ -62,14 +62,15 @@ MASTER_COMMIT="$(git rev-parse HEAD)"
 
 git checkout -b gh-pages upstream/gh-pages || exit 1
 
-git rm -rf nightly || true
-rm -rf nightly || true
+OLD_NIGHTLIES="$(find nightly -type f -mtime +1)"
```

Comment:
Oh ugh.  Should we do instead `find ... -exec git rm -rf {} + -exec rm -rf {} +`?